deployments/gpu-operator/values.yaml

```diff
@@ -77,6 +77,7 @@ operator:
   use_ocp_driver_toolkit: false
   # cleanup CRD on chart un-install
   cleanupCRD: false
+  imageCleanupCRD: bitnami/kubectl:1.31
```

Comment:
We can use the upstream `bitnami/kubectl` image here as we don't need the CRDs in the container for the cleanup task

Reply:
The cleanest approach would be to add a `DeleteCRD` method to the k8s-operator-libs. I think we can revisit this PR after that. Thoughts?

Reply:
Good call, was also thinking about this. Let me add a command to delete CRDs to the cmdutil

Reply:
There may be other requirements in terms of using a thirdparty image. Implementing the required logic in code may be simpler in the long run.